scripts/assessment-cof-reporting_scores.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
+	sub_criteria_id AS "Score Subcriteria",
+	score AS "Score",
+	justification AS "Score Justification"
+FROM
+	scores
```

Comment:
As I understand it this is just getting all the scores? How are they linked to the application ID? Also subcriterias can be 're-scored' so we need to ensure we're only taking the latest in each case.